src/portfft/descriptor.hpp

```diff
@@ -191,6 +191,12 @@ class committed_descriptor {
       IdxGlobal n_transforms, IdxGlobal batch_start, Idx total_factors, IdxGlobal output_offset, sycl::queue& queue,
       const std::vector<sycl::event>& events, complex_storage storage);
 
+  /**
+   * vector containing the implementation level selected, and the sub-implementation level, factors, and kernel_ids
+   * associated with each of the factor.
```

Comment:
```suggestion
   * Vector containing the sub-implementation level, kernel_ids and factors for each factor that requires a separate kernel.
```
That documentation is plain wrong!

Reply:
You only fixed this partly. Capitalization is still missing, documentation is still in different order compared to tuple elements. And it would still be nice to add that this is for the factors that require separate kernels, not the factors within individual kernel.

Reply:
Fixed

Reply:
Following Tadej's suggestion you should rephrase `and the factors associated with each of the factor` to `and factors for each factor that requires a separate kernel` to resolve this.

Reply:
I think this is done with https://github.com/codeplaysoftware/portFFT/pull/134/commits/de7aad456cc76feec8a2fb9b2abf6bbfc8f7ca02 now